vunit/activehdl_interface.py

```diff
@@ -26,7 +28,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class ActiveHDLInterface(SimulatorInterface):
+class ActiveHDLInterface(VsimSimulatorMixin, SimulatorInterface):

```

Comment:
Why is it necessary to inherit from VsimSimulatorMixin? I think that could be a misunderstanding.

Reply:
It probably is a misunderstanding as it's a bit of a challenge to jump into such a big project. The assumption was that akin to Riviera Pro and Modelsim, ActiveHDL is also a vsim based simulator and hence would require this inheritance.  I have just tested the performance without this inheritance and everything seems to runs as it was before, so will update as a part of the next commit.